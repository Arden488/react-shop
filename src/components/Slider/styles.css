.main-slider {
  max-width: 100%;
  height: 600px;
  text-align: center;
  overflow: hidden;
  position: relative;
}

.main-slider__item {
  display: none;
  position: absolute;
  top: 0;
  left: 50%;
  transform: translateX(-50%);
}

.main-slider__item_active {
  display: block;
}
