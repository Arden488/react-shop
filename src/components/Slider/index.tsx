import * as React from 'react';

import './styles.css';

const Slider = () => (
  <div>
    Slider
  </div>
);

export default Slider;
