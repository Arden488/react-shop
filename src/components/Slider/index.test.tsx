import { shallow } from 'enzyme';
import * as React from 'react';

import Slider from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<Slider />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
