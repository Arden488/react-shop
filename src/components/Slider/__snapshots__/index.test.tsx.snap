// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main-slider"
>
  <img
    alt=""
    className="main-slider__item main-slider__item_active"
    src="img/slide3.jpg"
  />
  <img
    alt=""
    className="main-slider__item"
    src="img/slide2.jpg"
  />
  <img
    alt=""
    className="main-slider__item"
    src="img/slide1.jpg"
  />
</div>
`;
